### ISSUE TITLE
Move SEO panel to a different location

### ISSUE THREAD

**Issue by SnowCreative** [Sterc/SEOSuite#51]
The SEO panel is really large, pushing the Content area way down the screen. It would be great to be able to move this under the Content area, or into a tab by itself.

**Comment by BigBlockStudios** [comment]
I agree - it needs to be in it's own tab.

**Comment by DESIGNfromWITHIN** [comment]
yes would like to request this also.

**Comment by dimasites** [comment]
Hi! This is very usefull feature, and may works like MODX system setting [[++tvs_below_content]]
and in my dreams i see this order if [[++tvs_below_content]] and [[++seosuite_below_content]] optioned in Yes:
1. Content panel
2. TVs panel
3. SEOsuite panel

**Comment by pepebe** [comment]
+1 from me as well.

**Comment by rdegler** [comment]
Yes, this would really help.  Content editors are completely thrown off and having the content pushed off the screen.  Please allow the SEOsuite panel to be below the content or in a tab of its own.

Thanks,
Roy

**Comment by sfleif** [comment]
In some installations the SEOsuite stuff is below the content and in some above. How is that configurated?

**Comment by VADsky** [comment]
на скорую руку использую Plugin:


```php
<?php
// OnDocFormRender
$modx->controller->addHtml('<script>Ext.onReady(function(){
	setTimeout(function(){
        hideseo = document.querySelector("div#resource-seosuite-panel > div > div.x-tool.x-tool-toggle");
        hideseo.click();
	}, 300);
});</script>');
```

**Comment by crimsonpixel** [comment]
I have inherited a site where the SEO Panel appears above the tabs which I think really focuses site managers to fill in these fields.  The bottom of the page doesn't always work.  I can see there is an option to now have it above the content but this is within the content section and sometimes this section is hidden for users if TVs are the main source of input.

I would love to position the SEO panel like this on other sites but I cannot find out how this was implemented!

![Image](https://github.com/user-attachments/assets/0765bb18-472f-4176-96f2-669babc10fd7)

**Comment by dimasites** [comment]
@crimsonpixel i'am afraid, you inherited the website with old SEOtab addon, but not SEOsuite.

P.S. Moving SEO panel ability in SEOsuite is stop-factor for use this plugin for me, and i use (auto)Redirector with some TVs for meta-tags... as temporary solution. @gpsietzema and all contributors, please know that we are waiting for an update! Thx for attantion!

**Comment by rdegler** [comment]
Any possibility of this ever being implemented

**Comment by gpsietzema** [comment]
@rdegler as far as I know this is already the case for some years. Try this setting:

<img width="1733" height="118" alt="Image" src="https://github.com/user-attachments/assets/0540f0a8-ee99-46bb-8a40-e086365c6039" />
